resources/analysisTools/copyNumberEstimationWorkflow/bashLib.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Copyright (c) 2017 The ACEseq workflow developers.
+# Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/ACEseqWorkflow/LICENSE.txt).
+
+dieWith() {
+  local ec="${2:-$?}"
+  local msg="${1:?No error message}"
```

Comment:
```suggestion
  local msg="${1:?No error message}"
  local ec="${2:-$?}"
```

better keep them ordered